app/uk/gov/hmrc/trusts/controllers/GetEstateController.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.trusts.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent}
+import uk.gov.hmrc.play.bootstrap.controller.BaseController
+import uk.gov.hmrc.trusts.controllers.actions.{IdentifierAction, ValidateUTRAction}
```

Comment:
Maybe something to refactor later, I'd like to keep Identifier and Validate the same in the sense they both have an `apply()` function and use DI to inject them into the controller.